[IMP] l10n_latam_check_ux: Added restriction on checks delete

feg-adhoc opened the pull request.

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/659).

jjscarafia committed:
[IMP] payment_pro: send default to pay amount

Esto es necesario porque ahora en pagos de clientes el onchage de withholdings no computa amount
Además, sin l10n_Ar_tax no estamos mandaando el amount

Quedaría una mejora "linda" que sería evaluar si el default journal que se va a usar es en otra moneda, tal vez deberíamos convertir el importe que se manda a esa otra moenda (pero la verdad no parece ser un caso necesario por el momento)

closes ingadhoc/account-payment#765

Related: ingadhoc/odoo-argentina#1189
Signed-off-by: Camila Vives <cav@adhoc.inc>

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/659.png)](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/659)

roboadhoc commented:
@feg-adhoc I didn't know about this PR and had to retrieve its information, you may have to re-approve it as I didn't see previous commands.
